app/scripts/metamask-controller.js

```diff
@@ -4646,6 +4647,27 @@ export default class MetamaskController extends EventEmitter {
     return keyring.isUnlocked();
   }
 
+  /**
+   * get hardware wallet Device name for metric logging in UI.
+   * Currently, it only handle the special case for OneKeyDevice connect metamask via Trezor.
```

Comment:
IMO, we should not add details in the doc itself, WDYT?

```suggestion
   * get preferred hardware wallet device name.
```